euphoria-operator-testkit/src/main/java/cz/seznam/euphoria/operator/test/ReduceByKeyTest.java

```diff
@@ -337,13 +339,52 @@ public void testReduceWithoutWindowing() {
     });
   }
 
+  @Processing(Processing.Type.BOUNDED)
```

Comment:
This looks odd, why not reduce the sorted values to list and check the final list?